packages/fx-core/templates/core/v3Migration/js.ts.app.local.yml

```diff
@@ -124,11 +124,17 @@ deploy:
       {{#dotnet}}
       dotnet: true
       {{/dotnet}}
-    {{#if devCert}}
-    # Output: following environment variable will be persisted in current environment's .env file.
-    # SSL_CRT_FILE: certificate file
-    # SSL_KEY_FILE: certificate key
-    {{/if}}
+    writeToEnvironmentFile: # Write the information of installed dependencies into environment file for the specified environment variable(s).
+      {{#devCert}}
+      sslCertFile: SSL_CRT_FILE
+      sslKeyFile: SSL_KEY_FILE
+      {{/devCert}}
+      {{#func}}
+      funcPath: FUNC_PATH
+      {{/func}}
+      {{#dotnet}}
+      dotnetPath: DOTNET_PATH
```

Comment:
The `cli/runDotnetCommand` sets `execPath: {{execPath}}` (see line 146), should the env var name also be set to `execPath`? The goal is avoid mismatch when the logic changes one place but forgets another place.

Reply:
Also need to see whether FUNC_PATH needs to do same thing

Reply:
@kuojianlu  please help check it. The output in prerequisite/install and the input in cli/runDotnetCommand

Reply:
The {{execPath}} in line 146 will be replaced with DOTNET_PATH by taskMigration migrateValidateLocalPrerequisites.
So the expected output will be  execPath: ${{DOTNET_PATH}}